### PR TITLE
Add change background feature

### PR DESCRIPTION
**Pull request by manfredt27**
This feature introduces a "Change background" button on top of every form, which is intended for the user to pick an image from his/her computer and load it as a background for his/her UVP.

ng-flow (https://github.com/flowjs/ng-flow) is needed for this feature
To install run:
`bower install`

### CODE REVIEW

**Comment by BrayanZ** [comment]
Can you extend more what the feature is about?

**Comment by manfredt27** [comment]
Like that?

**Comment by BrayanZ** [comment]
Yes, and also we don't need to run `bower install "ng-flow#~2" --save` to install `ng-flow`.
We just need to run `npm install` or `bower install` to update the dependencies, since when you ran that with the `--save` option (adds the package to the dependencies list of the project)

File: src/app/forms/forms.module.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+      flowFactoryProvider.defaults = {
+        target: 'app/forms/upload.php',
+        permanentErrors: [404, 500, 501],
+        maxChunkRetries: 1,
+        chunkRetryInterval: 5000,
+        simultaneousUploads: 4,
+        singleFile: true
+      };
+      flowFactoryProvider.on('catchAll', function (event) {
+        console.log('catchAll', arguments);
```
Comment (BrayanZ) [inline src/app/forms/forms.module.js:15]: remove the console logs.

File: src/app/forms/forms.module.js [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+    .config(['flowFactoryProvider', function (flowFactoryProvider) {
+      flowFactoryProvider.defaults = {
+        target: 'app/forms/upload.php',
+        permanentErrors: [404, 500, 501],
+        maxChunkRetries: 1,
+        chunkRetryInterval: 5000,
+        simultaneousUploads: 4,
+        singleFile: true
+      };
+      flowFactoryProvider.on('catchAll', function (event) {
```
Comment (BrayanZ) [inline src/app/forms/forms.module.js:14]: besides the log, you're doing nothing when this event occurs

File: src/app/forms/forms.module.js [hunk 3]
```diff
@@ -1,5 +1,21 @@
 (function() {
   'use strict';
 
-  angular.module('uvpBuilderWeb.forms', []);
+  angular.module('uvpBuilderWeb.forms', ['flow'])
+    .config(['flowFactoryProvider', function (flowFactoryProvider) {
+      flowFactoryProvider.defaults = {
```
Comment (BrayanZ) [inline src/app/forms/forms.module.js:6]: Do we really need all this configurations?

File: src/app/forms/forms.module.js [hunk 4]
```diff
@@ -1,5 +1,21 @@
 (function() {
   'use strict';
 
-  angular.module('uvpBuilderWeb.forms', []);
+  angular.module('uvpBuilderWeb.forms', ['flow'])
+    .config(['flowFactoryProvider', function (flowFactoryProvider) {
```
Comment (BrayanZ) [inline src/app/forms/forms.module.js:5]:
``` javascript
.config(flowFactoryConfig);

flowFactoryConfig.$inject= ["flowFactoryProvider"];

function flowFactoryConfig(flowFactoryProvider) {
  ...
}

```

File: src/app/forms/methods.controller.js [hunk 5]
```diff
@@ -23,5 +25,11 @@
     function back() {
       vm.showUVP = false;
     }
+
+    $scope.$on('flow::complete', function (event, $flow) {
+      userBackground.image = $flow.files[0];
+      console.log(userBackground.image);
```
Comment (BrayanZ) [inline src/app/forms/methods.controller.js:31]: remove logs

**Pull request closed without merging**